lib/phoenix_pubsub_redis/redis_server.ex

```diff
@@ -67,6 +70,11 @@ defmodule Phoenix.PubSub.RedisServer do
     {:noreply, state}
   end
 
+  def handle_info({:redix_pubsub, redix_pid, :disconnected, %{reason: reason}}, %{redix_pid: redix_pid} = state) do
+    Logger.error "disconnected from pubsub: #{reason}"
+    {:noreply, state}
```

Comment:
Let's make the error:

    Logger.error "[Phoenix.PubSub.Redis] disconnected from redis. Awaiting reconnection..."